package.json

```diff
@@ -19,7 +19,7 @@
     "publish": "lerna publish from-package --ignore-scripts",
     "watch": "run-p --print-label watch:*",
     "lint": "run-p --print-label lint:*",
-    "start:docker": "docker-compose up mongo mongo-express",
+    "start:docker": "docker-compose up mongo mongo-express media",
```

Comment:
The idea is that you run `yarn run dev`. This will run `start:docker` and also the `watch` command which starts all the services like api, media, editor etc. on your local machine. If you want to start the whole environment in docker you can just do that with `docker-compose up`.